src/org/traccar/api/BaseResource.java

```diff
@@ -29,4 +37,35 @@ protected long getUserId() {
         }
         return 0;
     }
+
+    protected void handlePermission(Map<String, Long> entity, boolean link) throws SQLException {
+        if (entity.size() != 2) {
```

Comment:
I think we can remove this check. If someone misuses the API, he will either get an error or extra parameters will be ignored.